### DISCUSSION TITLE
[V5] Suspicious 'WARN' Log entry in production

### DISCUSSION THREAD

**Discussion by aayush123** [adonisjs/core#2618] [Help] [answered]
Hi,
So I have an Adonis backend running on AWS.
The backend uses a managed MySQL running on RDS (free tier as of now since we just started off).

The thing that concerns me is that I notice a frequent WARN log in pm2 logs (attached screenshot below):
The error message is `Connection Error: Error: Connection lost: The server closed the connection.`

![image](https://user-images.githubusercontent.com/5837750/118357768-58d65f80-b599-11eb-9a64-1ac07488f068.png)

As far as I believe, this is the RDS server closing old connections present in the Adonis pool (I may be wrong)

I'd be very grateful if anyone could help out on how to prevent this.
Thanks :)

**Comment by thetutlage** [comment, marked as answer]
This is something you have to check with the database server if it allows a persistent connection or not. If server drops the connection, then Lucid warns you about it

**Reply by aayush123** [reply]
Yep, it seems the default for connection timeout is 8 hours on RDS.
During my investigation, I also noticed that this log generally appears early in the morning, local time. So I guess connections get stale in the night and are dropped when Lucid tries to use them later in the morning, hence the warning.
Guess I'll just ignore it and hopefully it'll not appear once we start getting more traffic :)
